examples/proof-of-concept/aws-iot/docs/aws-iot-integration.puml

```diff
@@ -38,14 +43,19 @@ frame "Edge Tier" {
     }
 }
 
+
 am -left-> ds : Store\nThings
-idm -down-> ds
-idm <-> am
+idm -down-> ds : Store\nThings
+idm - am
 device -up-> am : Register &\nAuthorize
 device -up-> hub : Publish\n(Access Token)
 hub -> lamb : Authorize
+hub -up-> reg
 lamb -down-> service : Verify\nToken
 service -> am
+idm -up-> icf : Synchronise \n Things
+icf -right-> sdk
+sdk -right-> reg : Query \n Things
```

Comment:
Since FR can modify identities in AWS is `Sync` better than `Query`
I need to do this for the Azure\GCP PoCs as well.

Reply:
Good point. I'll change it for the others as well.